dataline-workers/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
     implementation project(":dataline-integrations")
 
     testImplementation "org.mockito:mockito-inline:2.13.0"
-    testImplementation "org.postgresql:postgresql:42.2.16"
     testImplementation "org.testcontainers:testcontainers:1.14.3"
     testImplementation "org.testcontainers:postgresql:1.14.3"
+    testImplementation "org.postgresql:postgresql:42.2.16"
```

Comment:
isn't it better when it is sorted?

Reply:
is there a plugin that will sort this?

Reply:
Found [this](https://github.com/platan/idea-gradle-dependencies-formatter). Might toy around with it but generally I don't find myself spending any meaningful time reorganizing deps